solidity/ecdsa/contracts/WalletRegistry.sol

```diff
@@ -772,6 +772,47 @@ contract WalletRegistry is
         return dkg.currentState();
     }
 
+    /// @notice Checks whether the given operator is a member of the given
+    ///         wallet signing group.
+    /// @param walletID ID of the wallet
+    /// @param walletMembersIDs Identifiers of the wallet signing group members
+    /// @param operator Address of the checked operator
+    /// @param walletMemberIndex Position of the operator in the wallet signing
+    ///        group members list
+    /// @return True - if the operator is a member of the given wallet signing
+    ///         group. False - otherwise.
+    /// @dev Requirements:
+    ///      - The `operator` parameter must be an actual sortition pool operator.
+    ///      - The expression `keccak256(abi.encode(walletMembersIDs))` must
+    ///        be exactly the same as the hash stored under `membersIdsHash`
+    ///        for the given `walletID`.
+    ///      - The `walletMemberIndex` must be in range [0, walletMembersIDs.length-1]
```

Comment:
We use a convention that group member index is between [1, groupSize] given it's what needs to be used during the off-chain DKG. See examples:
https://github.com/keep-network/keep-core/blob/09f9c6b1761622757b21cd9829b3eeaadc8bd21b/solidity/ecdsa/contracts/libraries/EcdsaDkg.sol#L97-L99

https://github.com/keep-network/keep-core/blob/main/solidity/ecdsa/contracts/libraries/EcdsaDkg.sol#L83-L86

https://github.com/keep-network/keep-core/blob/main/solidity/ecdsa/contracts/libraries/EcdsaDkg.sol#L78-L80

It means we should accept a value in range [1, groupSize] and then just `-1`on `walletMembersIDs[walletMemberIndex - 1] == operatorID`

Reply:
Yep, right! See https://github.com/keep-network/keep-core/pull/2925/commits/9904a69bcf49963814cde94b91042423819fd333